src/AppBundle/Resources/views/App/component.html.twig

```diff
@@ -0,0 +1,124 @@
+{% extends "AppBundle::layout.html.twig" %} 
```

Comment:
It will be better to split the left panel from the rest. For example a `sidebar.html.twig`